FoodBookApp/UI/Views/CreateReview/CreateReview1View.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  CreateReviewView.swift
+//  FoodBookApp
+//
+//  Created by Laura Restrepo on 28/02/24.
+//
+
+import Foundation
+import SwiftUI
+
+struct CreateReview1View: View {
+    @State private var model = CreateReview1ViewModel()
+    @State private var searchText: String = ""
+    @FocusState private var searchTextIsFocused: Bool
+    @State private var selectedCats: [String] = []
+    let customGray = Color(red: 242/255, green: 242/255, blue: 242/255)
```

Comment:
not urgent not blocker but these colors could be in a constants or theme file later on.